internal/cmd/generate.go

```diff
@@ -30,6 +30,12 @@ func (c *Generate) Execute(opts *commander.CommandHelper) {
 		return
 	}
 
+	mustFollow := false
+	follow := opts.Arg(2)
```

Comment:
Please add it as a flag and not a positional argument.

Example:
* Configure: https://github.com/yitsushi/totp-cli/blob/main/internal/cmd/set_prefix.go#L91-L93
* In action: https://github.com/yitsushi/totp-cli/blob/main/internal/cmd/set_prefix.go#L125-L127

Reply:
Thank you for pointing me out to the flag possibility. It is much better.